test/extensions/tracers/opentelemetry/samplers/always_on/always_on_sampler_test.cc

```diff
@@ -0,0 +1,55 @@
+#include <string>
+
+#include "envoy/extensions/tracers/opentelemetry/samplers/v3/always_on_sampler.pb.h"
+
+#include "source/extensions/tracers/opentelemetry/samplers/always_on/always_on_sampler.h"
+#include "source/extensions/tracers/opentelemetry/span_context.h"
+
+#include "test/mocks/server/tracer_factory_context.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+TEST(AlwaysOnSamplerTest, TestWithInvalidParentContext) {
```

Comment:
They asked me to add one-line comment describing what the test does
```suggestion
// Verify sampler being invoked with an invalid span context
TEST(AlwaysOnSamplerTest, TestWithInvalidParentContext) {
```